hazelcast-spring/src/main/resources/hazelcast-spring-3.9.xsd

```diff
@@ -1471,7 +1471,7 @@
         <xs:complexType>
             <xs:complexContent>
                 <xs:extension base="hazelcast-bean">
-                    <xs:all>
+                    <xs:choice minOccurs="1" maxOccurs="unbounded">
```

Comment:
Hmm, so basically the `<xs choice minOccurs="1"` has no effects, since a e.g. even an absent Near Cache configuration (which is allowed by its own `minOccurs="0"`) counts to the choice. So we cannot enforce that "at least one child element needs to be defined" if there are optional elements like the Near Cache.

So this works fine, but it still looks like it should do something else :) Wouldn't be `<xs choice `minOccurs="0"` be more logical then? Even now a total empty block should work fine, doesn't it?